docs/rules/no-node-access.md

```diff
@@ -0,0 +1,44 @@
+# Disallow direct Node access (no-node-access)
+
+The Testing Library already provides methods for querying DOM elements.
+
+## Rule Details
+
+This rule aims to disallow DOM traversal using native HTML methods and properties, such as `closest`, `lastChild` and all that returns another Node element from an HTML tree.
+
+Examples of **incorrect** code for this rule:
+
+```js
+screen.getByText('Submit').closest('button'); // chaining with Testing Library methods
+```
+
+```js
+const buttons = screen.getAllByRole('button');
+const buttonA = buttons[1]; // getting the element directly from the array
+
+expect(buttonA.lastChild).toBeInTheDocument();
+```
+
+```js
+const buttonText = screen.getByText('Submit');
+const button = buttonText.closest('button');
+```
+
+```js
+document.getElementById('submit-btn').closest('button');
+```
+
+Examples of **correct** code for this rule:
+
+```js
+const buttonText = screen.getByRole('button');
+expect(buttonText.textContent).toBe('Submit');
+```
```

Comment:
I see all the examples uses `screen`. Though it's the recommended way, there's another query that considers that; so for the scope of this one, using destructuring with render should work as well.

Can we add those examples here in the readme, and also in the tests?

Reply:
Nice! I thought about those and forgot to add them, thanks.